docs/quickstart-aws.md

```diff
@@ -189,14 +194,14 @@ returning an error such as:
 ```
 The machinedeployments "pool1" is invalid: metadata.resourceVersion: Invalid value: 0x0: must be specified for an update
 ```
-For a workaround, please follow the steps described in the [issue 593][scale_issue].
+For a workaround, please follow the steps described in the [issue 593][scale_issue]. But the above will work with `kubectl` 1.16.x.
```

Comment:
I'd combine this into one sentence such as:
```suggestion
For a workaround, please follow the steps described in the [issue 593][scale_issue] or upgrade to `kubectl` 1.16 or newer.
```

Reply:
Fixed. :slightly_smiling_face: